### ISSUE TITLE
btt skr mini e3 v3 plus btt tft35 e3 v3 wierd temperature stuff

### ISSUE THREAD

**Issue by sputnik333** [bigtreetech/BTT-TFT35-E3-V3.0#95]
So i have a btt tft35 e3 v3  screen with the "BIGTREE_GD_TFT35_V3.0_E3.27.x" update on it, on a btt skr mini e3 v3 running marlin bugfix 2.1.x and i think that there is something wrong... when i print the temperature has a life on its own. the set temperature rises and falls like it is commanded to do so, but the gcode is not telling it to do it.

I have added a link to a onedrive folder where i have 2 vids and a few fotos in that hopefully describes my problem (high volume warning for headset users). I have allso added my entire build folder in there - so if it can help anything it is there.

The screen is doing it in marlin mode and in tft mode. It seems, and i might be wrong here, like the printer behaves a bit better when starting in marlin mode - changeing to tft mode - and then back to marlin mode.... that might just be me thinking that though...
PID autotune on nozzle and bed has been run with a cycle count of 10 just because.

the setup on the pid i have in my settings files has calmed it down a bit, but it takes forever to heat up now because it isn't allowed 100% power to heat up nozzle...

so to recap

Set print temp changes "at will" - changeing modes (maybe) helps somewhat.

In that folder i will also add my config files.

The printer is an first gen ender 3.

link to folder is:

https://1drv.ms/f/s!AkRqDh64SZEdh-Re4M8LemU54p9vlA?e=OY5oaa

**Comment by sputnik333** [comment]
nevermind i was an idiot.. i had activated auto temp thingy in marlin...
